test/index.test.ts

```diff
@@ -196,7 +196,7 @@ describe('run', function () {
           'ncu-mock-pre': '1.0.0'
         }
       }),
-      target: 'newest'
+      target: () => 'newest'
```

Comment:
You'll want to leave these tests as-is. It should support static target or function target, just like `filter`.

Reply:
Since it should support a static target as well there is actually no big benefit to set it to a function in the initOptions then, or? 🤔